src/configure-nilrt-snac

```diff
@@ -140,6 +142,26 @@ remove_niauth() {
 	trap - EXIT
 }
 
+# Install and configure pam-plugin-faillock.
+# Any non-root account will get locked after 3 failed authentications within 15 minutes.
+configure_faillock() {
+	log INFO Configuring faillock...
+
+	log DEBUG Installing pam-plugin-faillock...
+	opkg install pam-plugin-faillock
+
+	log DEBUG Configuring faillock settings...
+	sed -i 's/^# audit$/audit/' "${FAILLOCK_CONF}"
+	sed -i 's/^# silent$/silent/' "${FAILLOCK_CONF}"
+	sed -i 's/^# deny = 3$/deny = 3/' "${FAILLOCK_CONF}"
```

Comment:
I'm curious why you specify this even though the line above it (in the description) says 3 is the default. There are other default values (such as the log location) that you didn't override.

Reply:
I'm just mimicking the v0.1 instructions, where we instruct the user to uncomment that line. I don't have a strong opinion on whether we should specify the setting or rely on the default.